src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -77,24 +129,32 @@ public sealed class PeerConnector : IDisposable
             this.MaximumNodeConnections = 8;
```

Comment:
i know this is little offtopic at this PR, but i'd prefer a well documented constant instead of hardcoded value

but again, no need for that in this particular PR, just a random comment to consider later

Reply:
Correct but I am keen top push this (as generally this looks good) so @fassadlr future PR